app/2.0/docs/devguide/dom-template.md

```diff
@@ -141,12 +127,41 @@ class MyElement extends Polymer.Element {
 customElements.define('my-element', MyElement);
 ```
 
-### Inherited templates {#inherit}
+#### Return a string template (all 2.x versions) {#stringtemplate}
+Template literals are denoted by being enclosed in back-ticks (<code>` `</code>). 
+
+Specify a string template in a static template getter: { .caption }
+
+```js
+class MyElement extends Polymer.Element {
+
+  static get template() {
+    return `<style>:host { color: blue; }</style>
```

Comment:
Doesn't this need to be:

```js
  return html`<style>:host { color: blue; }</style>
```

Reply:
@justinfagnani per @arthurevans' feedback, I left that example in for users of pre-2.4 versions. 

I may have misunderstood which forms are available in which versions, though. My understanding was that in 2.4+, the string template return value is deprecated but still usable. For that reason the heading indicates that the example applies to "all 2.x versions" - would you mind clarifying on that point?

@arthurevans WDYT?